banzai/bias.py

```diff
@@ -140,6 +140,63 @@ def do_stage(self, images):
         return images
 
 
+class BiasComparer(ApplyCalibration):
+    # In a 16 megapixel image, this should flag 0 or 1 pixels statistically, much much less than 5% of the image
+    SIGNAL_TO_NOISE_THRESHOLD = 6.0
+    ACCEPTABLE_PIXEL_FRACTION = 0.05
+
+    def __init__(self, pipeline_context):
+        super(BiasComparer, self).__init__(pipeline_context)
+
+    @property
+    def group_by_keywords(self):
+        return ['ccdsum']
+
+    @property
+    def calibration_type(self):
+        return 'bias'
+
+    def on_missing_master_calibration(self, logging_tags):
+        self.logger.warning('No master Bias frame exists. Assuming these images are ok.', logging_tags)
+
+    def apply_master_calibration(self, images, master_calibration_image, logging_tags):
+        # Short circuit
+        if master_calibration_image.data is None:
+            return images
+
+        images_to_reject = []
+
+        for image in images:
+
+            # If the the fraction of pixels that deviate from the master by a s/n threshold exceeds an acceptable fraction
+            bad_pixel_fraction = np.abs(image.data - float(master_calibration_image.header['BIASLVL']) - master_calibration_image.data)
```

Comment:
Might be cleaner to create a method `get_bad_pixel_fraction` and put all this logic in there.